views/index.ejs

```diff
@@ -1,14 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
-<head> <%-include("./partials/head.ejs")%> </head>
+<head>
+    <%- include("./partials/head.ejs") %>
+</head>
 <body>
-<header> <%-include("./partials/header.ejs")%> </header>
-
-<p>'ello t' raveller</p>
-<p>beauteh</p>
-<% artists.forEach((artist)=> { %>
-    <h4><%= artist.name %></h4>
-    <h6> <%= artist.song %></h6>
-    <% });  %>
+<header> <%- include("./partials/header.ejs") %> </header>
+
+<div class="flex-container">
+    <div id="section" class="most-played-song-section">
+        <h2>Your most played song is:</h2>
+        <h4>Deadmau5</h4>
+        <h6>Ghosts 'n' stuff</h6>
+        <img class="most-played-images"
+             src="https://upload.wikimedia.org/wikipedia/en/d/d2/Deadmau5_and_Rob_Swire_-_Ghosts_N_Stuff_cover_art.jpg">
+    </div>
+
+    <div id="section" class="most-played-artist-section">
+        <h2>Your most played artist is:</h2>
+
+        <div class="most-played-artist-section-content">
+            <h4>Deadmau5</h4>
+            <img class="most-played-images"
+                 src="https://upload.wikimedia.org/wikipedia/en/d/d2/Deadmau5_and_Rob_Swire_-_Ghosts_N_Stuff_cover_art.jpg">
+        </div>
+    </div>
```

Comment:
could we move these sections into partials we extend upon? just to keep the code nice n modular